package.json

```diff
@@ -25,10 +25,11 @@
     "express-session": "^1.15.1",
     "grunt-cli": "^1.2.0",
     "grunt-pg": "https://github.com/watsoncj/grunt-pg.git",
-    "knex": "^0.12.9",
     "highcharts": "^5.0.12",
+    "jquery": "^3.2.1",
```

Comment:
you probably don't need this entire library if you are just making ajax calls axios or even fetch might do the trick